### ISSUE TITLE
Implement ra.session.describe('relations', ['unit', 'pilot'])

### ISSUE THREAD

**Issue by mturilli** [radical-cybertools/radical.analytics#21]

**Comment by andre-merzky** [comment]
Hey Matteo - this is basically asking for the inverse map of what we have, to obtain the pilot id by unit lookup, right?

**Comment by mturilli** [comment]
Yes, that's how I implemented it. I thought that as we use a tuple, it would make sense to offer a consistent implementation.

**Comment by mturilli** [comment]
This is how I use it so far:
https://github.com/radical-cybertools/radical.analytics/blob/experiment/swift/use_cases/rp_on_osg/bin/wrangler.py#L273
